src/applications/claims-status/containers/AppealInfo.jsx

```diff
@@ -21,11 +21,13 @@ import {
   APPEAL_TYPES,
   EVENT_TYPES,
   isolateAppeal,
-  RECORD_NOT_FOUND_ERROR,
-  AVAILABLE,
   getTypeName,
 } from '../utils/appeals-v2-helpers';
 
+import { RECORD_NOT_FOUND_ERROR } from '../actions/types';
+
+const AVAILABLE = 'AVAILABLE';
```

Comment:
Should this be added to the constants file, since it's used in 2 places?

Reply:
I'm not sure that this should even exist, I'm planning on getting to it in another round (this is round 7 lol). Good catch though